network_function/dockerfile

```diff
@@ -0,0 +1,11 @@
+# Specify the base image
+FROM gcc:latest
+# Copy the program files
+COPY nf.c /app/
+# Set the working directory
+WORKDIR /app
+# Compile the program
+RUN gcc -o nf nf.c
+# Set the entry point
+RUN chmod +x nf
```

Comment:
You can append this command with && into previous single RUN directive to reduce one additional layer in the container image.

Reply:
changed